extra/alacritty.man

```diff
@@ -57,6 +57,9 @@ Defines the window position. Falls back to position specified by window manager
 \fB\-t\fR, \fB\-\-title\fR <title>
 Defines the window title [default: Alacritty]
 .TP
+\fB\-\-embed\fR <parent>
+Sets the XEmbed parent window
```

Comment:
The type we expect (decimal) should be documented, this is also the case for the `--help` output.

Reply:
In addition to @chrisduerr comment, the man entry should be consistent with `--help` output, but as I can see they differ a bit.

Reply:
This should (hopefully) be set in my last commit.

Reply:
Ok, so I looked into making things more resilient if the window ID is bogus or the window doesn't exist.  The X FAQ suggests that checking for the existence of the target isn't really recommended:

Subject: 121)  How do I check whether a window ID is valid?
My program has the ID of a window on a remote display. I want to check whether
the window exists before doing anything with it.

	Because X is asynchronous, there isn't a guarantee that the window would
still exist between the time that you got the ID and the time you sent an event
to the window or otherwise manipulated it. What you should do is send the event
without checking, but install an error handler to catch any BadWindow errors, 
which would indicate that the window no longer exists. This scheme will work 
except on the [rare] occasion that the original window has been destroyed and 
its ID reallocated to another window.

Based on that, it seems like gracefully tolerating the issue is the most likely route to success.  It seems like std::panic::catch_unwind is the way this would be done in rust, but I don't see any examples of that in this codebase.  Before I try implementing it, I'd like to ask - Would using that meet your expectations?

Reply:
> The X FAQ suggests that checking for the existence of the target isn't really recommended

That doesn't mean we can't handle the errors. There's nothing garceful about the current approach, we just crash with a bogus error. `catch_unwind` should basically never used, since this is Rust we don't catch errors, we handle them.

This can simply be resolved by handling the `BadWindow` response that X will give when attempting to embed with an invalid window and printing an appropriate error message when that is the case.

Reply:
K.  I'll try to figure out what that involves tonight.  Thanks for the guidance.

Reply:
No worries, I think this just involves one of the usual C error handling ways where you ask X11 for the last error and it'll tell you about it. I haven't looked too much into it but I think the X11 bindings have something like this.

Reply:
K.  I pushed a change to handle re-parenting failures.  I'm not sure what the deal with the result from XReparentWindow is, but it isn't meaningful (always 1, success or failure) and the error handler doesn't get fired when it fails.  I couldn't find any examples, even in C, of code making use of the result.  Nevertheless, attempting to get attributes from the window after reparenting does get the feedback needed to assert the success of the reparenting.